src/components/ContactDetails.vue

```diff
@@ -502,6 +509,12 @@ export default {
 		contact() {
 			return this.$store.getters.getContact(this.contactKey)
 		},
+
+		excludeFromBirthdayLabel() {
+			return this.localContact.vCard.hasProperty(this.excludeFromBirthdayKey)
+				? t('Include contact to Birthday Calendar')
```

Comment:
```suggestion
				? t('contacts', 'Include contact to Birthday Calendar')
```
The function requires the app name as first argument.

Reply:
Oops, sorry for the typo ! Just fixed it.